### ISSUE TITLE
[RFE] Use systemd D-Bus API's GetUnitByPID to get the service

### ISSUE THREAD

**Issue by tjanez** [FrostyX/tracer#80]
It would we nice if tracer used [systemd D-Bus API](https://www.freedesktop.org/wiki/Software/systemd/dbus/)'s `GetUnitByPID` method to get the service from PID.

I've tested this and it works nicely. For example, to get the service name for a Postfix's subprocess `tlsmgr` with PID `12457`, one can use the following `gdbus` command.

```bash
$ systemctl status postfix.service 
● postfix.service - Postfix Mail Transport Agent
   Loaded: loaded (/usr/lib/systemd/system/postfix.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2017-04-18 14:21:42 CEST; 5 days ago
 Main PID: 1838 (master)
   CGroup: /system.slice/postfix.service
           ├─ 1838 /usr/libexec/postfix/master -w
           ├─ 1871 qmgr -l -t unix -u
           ├─12457 tlsmgr -l -t unix -u
           └─57532 pickup -l -t unix -u

[ ... log removed ... ]
$ gdbus call --system --dest org.freedesktop.systemd1   --object-path /org/freedesktop/systemd1   --method org.freedesktop.systemd1.Manager.GetUnitByPID 12457
(objectpath '/org/freedesktop/systemd1/unit/postfix_2eservice',)
```

This would also fix issues like #74 easily.

**Comment by sean797** [comment]
@tjanez I created https://github.com/FrostyX/tracer/pull/82. If you have a chance to have a look I would appreciate it

**Comment by sean797** [comment]
Closing fixed in #82
